### DISCUSSION TITLE
Content imported from sourcing to consumption should not be linked to collection if it is not published

### DISCUSSION THREAD

**Discussion by amitpriyadarshi** [Sunbird-Knowlg/Community#29] [Bugs]
@vinukumar-vs 
Content imported from sourcing to consumption should not be linked to collection if it is not published.

Currently, if the content is imported from sourcing to consumption and content published is failed, even though it gets linked with the respective collection passed in the event.

cc: @AmiableAnil

**Comment by vinukumar-vs** [comment]
@AmiableAnil Is content import expected to import Draf content as well? If we block this, we not need to handle this kind of issues.

One more check, How can we link Draft content to the collection. This is not expected behavior. Then why it is failing for imported contents?

@maheshkumargangula, @pallakartheekreddy

**Comment by AmiableAnil** [comment]
@amitpriyadarshi @vinukumar-vs Currently when we are linking the content to a collection as part of bulk approval (content-auto-creator), status check is not there. This is a bug. We will be fixing this issue.

**Reply by amitpriyadarshi** [reply]
Thanks @AmiableAnil I will create the the ticket and let you know.
